### PR TITLE
Issue 166 render failure rate requirement

### PR DESCRIPTION
**Pull request by Kasmadei**

### CODE REVIEW

**Comment by Kasmadei** [comment]
Not shure if also need to modify ctx object in eventModel.tsx

File: README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ It is primarily focused on FTA which offers possibilities for system partonomy d
 ```shell script
 npm install
 npm run build
-npm run preview
+npm run preview (only preview)
+npm run dev (for development)
```
Comment (blcham) [inline README.md:14]:
```suggestion
```

Comment (blcham) [reply README.md:14]: @Kasmadei 
We have "convention" in our projects to not mix development and production notes together but instead have them in separate sections of README. I will create a separate PR to fix the issue that you wanted to tackle with those proposals.

File: README.md [hunk 2]
```diff
@@ -10,7 +10,8 @@ It is primarily focused on FTA which offers possibilities for system partonomy d
 ```shell script
 npm install
 npm run build
-npm run preview
+npm run preview (only preview)
```
Comment (blcham) [inline README.md:13]:
```suggestion
npm run preview
```

**Review by blcham — APPROVED** [review]
LGTM

**Comment by blcham** [comment]
> Not shure if also need to modify ctx object in eventModel.tsx

I believe you should have.

**Pull request merged**